webapp/channels/src/components/user_settings/general/user_settings_general.tsx

```diff
@@ -1450,6 +1479,31 @@ export class UserSettingsGeneralTab extends PureComponent<Props, State> {
                 let extraInfo: JSX.Element|string;
                 let submit = null;
 
+                const validate = () => {
```

Comment:
Should we remove this validation from the submit function like we did with the email validation?

Reply:
I don't think this one is in the submit anymore